xla/service/gpu/gpu_compiler.cc

```diff
@@ -471,6 +471,7 @@ namespace {
 // Adds the HloVerifier for GPU to the given pipeline.
 void AddHloVerifier(HloPassPipeline* pipeline, HloVerifierOpts&& opts = {},
                     bool debug_only = false) {
+  opts.verify_unique_channel_ids = false;
```

Comment:
Since this is part of an attempt to remove the channel ID, can we base this on an experimental flag? Maybe `experimental_ignore_channel_ids`

Reply:
are you thinking to use the same flag to control when to ignore channel id in other places when we try to remove it? if so,  adding it makes sense.

Reply:
added the debug option